src/main/java/io/stargate/sgv2/jsonapi/service/schema/tables/ApiSupportDef.java

```diff
@@ -24,6 +24,15 @@ public interface ApiSupportDef {
    */
   boolean createTable();
 
+  /**
+   * The type can be used as component for collection
+   *
+   * <p>E.G. list values, set values, map keys, map values.
+   *
+   * @return <code>true</code> if the type can be used in a createTable statement.
+   */
+  CollectionSupport collectionSupport();
```

Comment:
this whole structure is exposed as support() normally, so lets just rename this property to "collections"  - we know we are talking about support